README.md

```diff
@@ -27,26 +27,66 @@ This test suite is designed to:
 ## Contributing
 See our [CONTRIBUTING.md](.github/CONTRIBUTING.md) for exact details on how to contribute.
 
-## Cromwell tests
+## Tests
 
-A suite of tests for running WDLs against Cromwell are located in `tests/cromwell-api`. Tests are run using Python. To run tests:
+There are two sets of tests for running WDLs against Cromwell using Python:
+
+- `tests/cromwelljava`: Runs WDLs on Cromwell using a local copy of `cromwell-<version>.jar` and `womtool-<version>.jar`. See "Java" below for details.
+- `tests/cromwellapi`: Runs WDLs on Cromwell running on FH clusters via the [PROOF API][proofapi]. See "API" below for details.
+
+What's the difference between the two sets of tests? The Java tests run against just the Java Cromwell code itself, whereas the API tests are a kind of integration test that incorporates all the infrastructure behind PROOF including the [PROOF API][proofapi], the Cromwell servers that PROOF manages, and the FH cluster infra that it all runs on.
+
+To run both sets of tests:
 
 - clone this repository locally and cd into it
 - if you don't have `uv` installed, [install it][uvinstall]
-- create a virtual environment: `uv venv`. This should pick up the Python version specified in `.python-version`. If you do not have the version given in `.python-version` it will be installed by `uv`. 
+- create a virtual environment: `uv venv`. This should pick up the Python version specified in `.python-version`. If you do not have the version given in `.python-version` it will be installed by `uv`
 - activate the virtual environment: `source .venv/bin/activate`
 - check that you have the Python version in `.python-version`: `python --version`
+
+Info:
```

Comment:
Can you be a little more descriptive here, like: "You should be somewhat familiar with pytest and uv:"

Reply:
yes can do